mod-director-core/src/main/java/net/jan/moddirector/core/configuration/ModDirectorRemoteMod.java

```diff
@@ -8,20 +9,31 @@
 import java.util.Collections;
 import java.util.Map;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
+
 public abstract class ModDirectorRemoteMod {
     private final RemoteModMetadata metadata;
     private final Map<String, Object> options;
+    private final String folderName;
```

Comment:
Currently this is named `folder` in the Json and `folderName` in the code. I think both are fine, but they should be the same in Json and code.

Reply:
Renamed.